gpflow/conditionals/uncertain_conditionals.py

```diff
@@ -59,60 +63,61 @@ def uncertain_conditional(Xnew_mu: tf.Tensor,
     if not white:
         q_mu = tf.linalg.triangular_solve(Luu, q_mu, lower=True)
         Luu_tiled = tf.tile(
-            Luu[None, :, :],
-            [num_func, 1, 1])  # remove line once issue 216 is fixed
+            Luu[None, :, :], [num_func, 1, 1]
+        )  # remove line once issue 216 is fixed
```

Comment:
comment now points to wrong line

Reply:
you just have to read "line" more generously...